builtin/logical/pki/path_tidy.go

```diff
@@ -41,6 +43,8 @@ var defaultTidyConfig = tidyConfig{
 	SafetyBuffer:       72 * time.Hour,
 	IssuerSafetyBuffer: 365 * 24 * time.Hour,
 	PauseDuration:      0 * time.Second,
+	MaintainCount:      true,
```

Comment:
Yeah, I'd maybe default to `false` and add a breaking change notice. Its less of a breaking change than was in the last release (wherein this was added defaulting to true, causing a lot of work on systems who aren't using metrics).

Alternatively, if we want to leave it to true, use a `*bool` value rather than `bool` and treat `nil==true`.